modules/core/pom.xml

```diff
@@ -164,6 +164,66 @@
             <version>1.0.0</version>
         </dependency>
 
+        <dependency>
```

Comment:
I'm not sure I agree we should add any dependencies to the ignite-core module. I suggest plugin to be developed (similar with ignite-direct-io).

Reply:
Yes. As I mentioned on dev-list - I don't have plans to add HTTP dependencies to the core module.
We should carefully discuss how and what exposers we want to integrate into product